src-docs/src/views/theme/typography/_typography_js.tsx

```diff
@@ -285,6 +287,29 @@ export const FontScaleValuesJS = () => {
           />
         </EuiDescribedFormGroup>
       </EuiPanel>
+      <EuiSpacer size="m" />
+      <EuiCallOut
+        iconType="accessibility"
+        title={
+          <>
+            We strongly recommend using relative (<EuiCode>rem</EuiCode> or{' '}
+            <EuiCode>em</EuiCode>) units instead of <EuiCode>px</EuiCode> when
+            possible{' '}
+          </>
+        }
+      >
+        <p>
+          Relative font units respect configured browser default font sizes,
+          which some users may set to larger than than the 16px default due to
+          visual impairment.{' '}
+          <EuiLink
+            href="https://usability.yale.edu/web-accessibility/articles/zoom-resizing-text"
+            target="_blank"
+          >
```

Comment:
Minor wordsmithing on this one. 

```suggestion
        <p>
          Relative font units respect configured browser default font sizes,
          which some users set larger than the 16px default.{' '}
          <EuiLink
            href="https://usability.yale.edu/web-accessibility/articles/zoom-resizing-text"
            target="_blank"
          >
```

Reply:
Can I ask why? Would it make more sense to say something like `due to, e.g. visual impairment or monitor sizes`? instead of specifically calling out visual ability?

Reply:
Yes, I like that suggestion to offer possible reasons outside just visual acuity.